docs/reference/announcements.md

```diff
@@ -4,6 +4,24 @@ title: "Announcements"
 description: "Important announcements including deprecation & removal notices"
 ---
 
+
+## Camunda 8.5
+
+
+### Support for Amazon OpenSearch
+As of the 8.5 release, Optimize is now also compatible with [Amazon OpenSearch](https://aws.amazon.com/de/opensearch-service/) 2.5.x. Note that using Amazon OpenSearch requires [setting up a new Camunda installation](/self-managed/platform-deployment/overview.md). A migration from previous versions or Elasticsearch environments is currently not supported.
```

Comment:
Do we want to mention that upgrade from older OpenSearch versions (i.e. 1.x) is currently not supported but will be supported in the future?

Reply:
@grlimacan, why will it be supported in the future? We are only aware of one customer using OpenSearch with Camunda 8, and we have found a customer solution for him.

Reply:
All right, then we don't need to worry about it. @RomanJRW I remember the request came from you that we should support the upgrade, was it about this customer? Is it still something we need to implement in the future or can we descope it completely?

Reply:
There are two things to consider:
- Upgrading from old versions of OpenSearch: We don't care about supporting this, we never do, even for ES. Customers manage their own DBs, we just say what is compatible
- Upgrading from Optimize 3.7 -> 3.13.x: We care about this. Customers are running 3.7 still with an old OS DB, and cannot move to newer Optimize versions until we offer official support. When we do, we need to provide an upgrade script that takes then from 3.7 schema to whatever schema they are on now